test/yarn-auditer.js

```diff
@@ -34,16 +34,15 @@ function testDir(s) {
 // eslint-disable-next-line func-names
 describe('yarn-auditer', function() {
   this.slow(3000);
-  it('prints yarn errors', () => {
+  it('prints unexpected https://registry.yarnpkg.com 503 error message', () => {
     const directory = testDir('yarn-error');
     const errorMessagePath = path.resolve(directory, 'error-message');
     const errorMessage = require(errorMessagePath); // eslint-disable-line
 
     return audit(
       config({
         directory,
-        registry: 'https://example.com',
```

Comment:
Note that the `--registry` flag doesn't work for `yarn audit` since it's not implemented yet. This is why I removed `registry: 'https://example.com'`.